partcad/src/partcad/cache.py

```
@@ -0,0 +1,94 @@
+
+    async def write_data_async(self, hash: CacheHash, items: dict[str, bytes]) -> dict[str, bool]:
+        """Write object to cache and return its hash."""
+        if not user_config.cache:
+            # Caching is disabled
+            return {}
+
+        cache_path = self.get_cache_path(hash)
+
+        saved = {}
+
+        async def task_item(key, value) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `key`

(ANN001)